lib/Basic/Statistic.cpp

```diff
@@ -126,6 +127,37 @@ auxName(StringRef ModuleName,
           + "-" + cleanName(OptType));
 }
 
+class UnifiedStatsReporter::RecursionSafeTimers {
+
+  struct RecursionSafeTimer {
+    llvm::Optional<SharedTimer> Timer;
+    size_t RecursionDepth;
+  };
+
+  StringMap<RecursionSafeTimer> Timers;
+
+public:
+
+  void BeginTimer(StringRef Name) {
```

Comment:
Nitpick: lowercase.

Reply:
Whoops, llvm-Caps-ism got the better of me.